extensions/database-documentation/src/common/utils.ts

```diff
@@ -0,0 +1,179 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as azdata from 'azdata'
+import * as vscode from 'vscode';
+// Language localization features
+import * as nls from 'vscode-nls';
+// The module 'openai' contains the OpenAI API
+import { Configuration, OpenAIApi } from "openai";
+
+const localize = nls.loadMessageBundle();
+
+export async function generateMarkdown(context: azdata.ObjectExplorerContext): Promise<string> {
```

Comment:
@aasimkhan30 Please make sure someone is reviewing this code - I won't have time to spend on this at the moment

(and ideally this is actually done in a separate PR, but I'll leave that up to you two)